sml_builder/templates/cookies.html

```diff
@@ -8,161 +8,249 @@
 {%- from "components/lists/_macro.njk" import onsList -%}
 
 {%- block main -%}
-    <div class="ons-page__body">
-        <h1 class="ons-u-fs-xxl">Cookies on the statistical methods library</h1>
-        {%- call onsPanel({
-            "classes": "ons-u-mb-m ons-u-d-no ons-cookies-confirmation-message",
-            "type": "success",
-            "iconType": "check",
-            "iconSize": "xl"
-        }) -%}
-            <h2 id="cookie_change_alert">Your cookie settings have been saved</h2>
-            <p class="ons-u-mb-no">Some parts of the ONS may use additional cookies and will have their own cookie policy and banner.</p>
-            <a class="ons-js-return-link ons-u-mt-s ons-u-dib" href="#0">Return to previous page</a>
-        {%- endcall -%}
+<div class="ons-page__body">
```

Comment:
Formatting has changed quite a bit here. Have you run the tools that Joe has implemented on the pipeline? I think black etc is pretty strict about tabbing ets

Reply:
Hi ran it and no error was highlighted in relation to this but i have tabbed it anyway :)